style_editor.html

```diff
@@ -1,4 +1,12 @@
+<!DOCTYPE html>
+
+<html lang="en">
 <head>
+ <title>Style Editor</title>
+  <meta name="description" content="ProffieOS Style Editor">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
What does this do?

Reply:
This should ensure it looks right on mobile devices.

Reply:
This should ensure it looks right on mobile devices.

Reply:
doesn't it also disable zooming?
(Which may make it very hard to use on mobile devices.)

Reply:
Zooming is fine, however I don't see it making a difference with or without. Again, it's been fine up to now, so there's no need I suppose.